src/Redux/api.tsx

```diff
@@ -634,19 +644,23 @@ const routes = {
   // States
   statesList: {
     path: "/api/v1/state/",
+    TRes: Type<IStateListResponse>(),
```

Comment:
Wrap it as `PaginatedResponse` inside this file! It'd be easier to read this way.

```suggestion
    TRes: Type<PaginatedResponse<StateModel>>(),
```